Change wording on appointment page

arcangelini opened the pull request:
## Changes proposed in this Pull Request

To help avoid confusion with what our scope of support is for quick start calls it seems best to remove the beginning sentence in the appointment confirmation page.

**Before**
[![Screenshot](https://d.pr/i/TvR991+)](https://d.pr/i/TvR991)

**After**
[![Screenshot](https://d.pr/rvxmH3+)](https://d.pr/rvxmH3)

## Testing instructions

* Go to http://calypso.localhost:3000/me/concierge
* Schedule an appointment
* Go back to http://calypso.localhost:3000/me/concierge
* View the statement below the top image

arcangelini committed:
Change wording on the appointment confirmation page. Removed - Wecan talk about anything related to your site -

jsnajdr commented on client/me/concierge/shared/appointment-info.js:
```diff
@@ -146,9 +146,7 @@ class AppointmentInfo extends Component {
-						'We can talk about anything related to your site. ' +
-							'Get all your questions ready ' +
-							'-- we look forward to chatting!',
+						'Get all your questions ready ' + '-- we look forward to chatting!',
```
While we're touching this, let's use a real mdash (—) character instead of the ASCII `--`.

Also, the `+` operator can be removed as the string is no longer that long and doesn't split over multiple lines.

jsnajdr reviewed (approved):
Looks good, suggesting some minor additional improvements to the affected code.

arcangelini committed:
Adjusted to be on one line and use actual m dash —

a8ci18n commented on client/me/concierge/shared/appointment-info.js:
```diff
@@ -145,17 +145,12 @@ class AppointmentInfo extends Component {
-							},
-						}
-					) }
+					description={ translate( 'Get all your questions ready — we look forward to chatting!', {
```
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'We can talk about anything related to your site. Get all your questions ready -- we look forward to chatting!' )` *ES Score: 7*

arcangelini commented:
> Looks good, suggesting some minor additional improvements to the affected code.

Thanks @jsnajdr I fixed the suggestion. I wasn't sure how `translate()` was working so I left it as-is. I also combined it and added the m dash. Am I good to merge and deploy this?

jsnajdr reviewed (approved):
> Am I good to merge and deploy this?

Yes, it looks good now :+1:

a8ci18n commented:
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6192451

Thank you @arcangelini for including a screenshot in the description! This is really helpful for our translators.

a8ci18n commented:
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6192451).
